webots_ros2_core/webots_ros2_core/devices/camera_device.py

```diff
@@ -36,13 +36,17 @@ class CameraDevice(SensorDevice):
     - Publishes raw image of type `sensor_msgs/Image`
     - Publishes intrinsic camera parameters of type `sensor_msgs/CameraInfo` (latched topic)
 
-    Args:
-        node (WebotsNode): The ROS2 node.
-        device_key (str): Unique identifier of the device used for configuration.
-        wb_device (Camera): Webots node of type Camera.
 
-    Kwargs:
-        params (dict): Inherited from `SensorDevice`
+    Parameters
+    ----------
+    node : WebotsNode
+      The ROS2 node.
+    device_key : str
+      Unique identifier of the device used for configuration.
+    wb_device : Camera
+      Webots node of type Camera.
+    params : dict
+      Inherited from `SensorDevice`
```

Comment:
I would remove the space before the colon:
```suggestion
    node: WebotsNode
      The ROS2 node.
    device_key: str
      Unique identifier of the device used for configuration.
    wb_device: Camera
      Webots node of type Camera.
    params: dict
      Inherited from `SensorDevice`
```